File: features/2_reactions/reactions.js
const Replies = require('/home/runner/kerfus-bot/funcs/pastas.js');

exports.run = async (msg,client) => {
  if(!msg.guild)return;
  const reacts = new Replies("reactions.json", client);
  for (var i =0;i< reacts.len(); i++){
    if (!reacts.includes(msg.formatted, i)) continue;
    if (!reacts.allowed(i, msg.guild.id, msg.channel.id)) continue;
    let reactions = reacts.get(i)
    for (const j of reactions)
      msg.react(j).catch(err=>{});
  }
}

File: features/2_reactions/reactions.py
from ...funcs import pastas as replies

async def run(msg, client):
    "Make the reactions using an object ?!!?!"
    if not msg.guild:
        return
    reacts = replies.Replies("reactions.json", client)
    for i in range(len(reacts)):
        if not reacts.includes(msg.formatted, i):
            continue
        if not reacts.allowed(i, msg.guild.id, msg.channel.id):
            continue
        reactions = reacts.get(i)
        for j in reactions:
            try:
                msg.react(j)
            except:
                pass
